bundle-edge.yaml

```diff
@@ -0,0 +1,43 @@
+bundle: kubernetes
+applications:
+  argo-controller:
+    charm: cs:~kubeflow-charmers/argo-controller
+    source: ./charms/argo-controller
+    scale: 1
```

Comment:
What makes this bundle "edge"; perhaps these charm definitions should include `channel: edge`?  Or is this instead a "lighter version of lite" bundle?

Reply:
Edge in the sense of edge deployments. This will be a stable bundle for edge deployment use cases. You're right that it conflicts with juju's concept of edge as a stability channel, but not sure of a better name.